common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -622,6 +739,13 @@ private static void incrementMemoryCounter(int capacity) {
     }
 
     private static void decrementMemoryCounter(int capacity) {
+        // Try to use the hook
+        if (_BITS != null) {
+            _BITS.unreserveMemory(capacity);
+            return;
+        }
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "usedMemory". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1854)